internal/core/openapi/legacy/api/apis/uc/uc_user_update_userinfo.go

```diff
@@ -91,6 +92,7 @@ func updateUserInfo(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	orgClient := r.Context().Value("org").(org.Interface)
```

Comment:
define a const for "org" as other provider dependencies

Reply:
use const defines at internal/core/openapi/legacy/component-protocol/protocol.go

Reply:
done